docker/base/Dockerfile.mysql57

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG bootstrap_version=26
+ARG bootstrap_version=27
```

Comment:
If we start changing this quite often, we'll probably want to add a minor version at some point.